pkg/test/e2eutil/wait_util.go

```diff
@@ -40,7 +44,7 @@ func WaitForOperatorDeployment(t *testing.T, kubeclient kubernetes.Interface, na
 	return waitForDeployment(t, kubeclient, namespace, name, replicas, retryInterval, timeout, true)
 }
 
-func waitForDeployment(t *testing.T, kubeclient kubernetes.Interface, namespace, name string, replicas int, retryInterval, timeout time.Duration, isOperator bool) error {
+func WaitForDeployment(t *testing.T, kubeclient kubernetes.Interface, namespace, name string, replicas int, retryInterval, timeout time.Duration, isOperator bool) error {
```

Comment:
This already exists a few lines above, on line 33, that's why your tests are failing.